e2e/catalogue/catalogue-test_1.md

```diff
@@ -0,0 +1,21 @@
+
+# Number
+
+1
+
+# Role
+
+Data manager of a network needs to check their network information.
+
+# Goal
+
+The data manager of the network Athlete needs to check following steps.
```

Comment:
Needs to check their network information, in order to .. / because ... ( without the why part is hard to evaluate the system response , i.e. the 'why' part is the goal